src/library.js

```diff
@@ -4186,12 +4229,6 @@ LibraryManager.library = {
   llvm_dbg_value: function() {},
   llvm_debugtrap: function() {},
   llvm_ctlz_i32: function() {},
-  emscripten_asm_const: function() {},
```

Comment:
This function no longer exists?   I guess it was never needed?

Reply:
Yes, it may have made sense in some old approach to EM_ASM in fastcomp perhaps.